### PR TITLE
bump identity and scanamo for snyk http4s issues and cats incompatibility

### PR DESCRIPTION
**Pull request by johnduffell**
This pulls in the new identity lib to prevent the http4s issue hitting us.
Also I had to bump scanamo due to a cats binary incompatibility issue.  That was probably overdue anyway.

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -6,7 +6,7 @@ object Dependencies {
   val awsClientVersion = "1.11.1022"
 
   val sentryLogback = "io.sentry" % "sentry-logback" % "1.7.5"
-  val identityAuth = "com.gu.identity" %% "identity-auth-play" % "3.235"
+  val identityAuth = "com.gu.identity" %% "identity-auth-play" % "3.248"
```
Comment (johnduffell) [inline project/Dependencies.scala:9]: waiting for https://github.com/guardian/identity/pull/1899 to ship

File: project/Dependencies.scala [hunk 2]
```diff
@@ -19,7 +19,7 @@ object Dependencies {
   val awsCloudWatch = "com.amazonaws" % "aws-java-sdk-cloudwatch" % awsClientVersion
   val membershipCommon = "com.gu" %% "membership-common" % "0.592"
   val scalaz = "org.scalaz" %% "scalaz-core" % "7.2.31"
-  val kinesis = "com.gu" % "kinesis-logback-appender" % "2.0.2"
+  val kinesis = "com.gu" % "kinesis-logback-appender" % "2.0.3"
```
Comment (johnduffell) [inline project/Dependencies.scala:22]: waiting for https://github.com/guardian/kinesis-logback-appender/pull/40 to ship

File: project/plugins.sbt [hunk 3]
```diff
@@ -3,7 +3,7 @@ logLevel := Level.Warn
 
 resolvers += Resolver.typesafeRepo("releases")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.7")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.9")
```
Comment (johnduffell) [inline project/plugins.sbt:6]: the guava bump is not present in 2.7.9 but hopefully will be available soon.

**Review by rupertbates — APPROVED** [review]
👍

File: project/Dependencies.scala (hunk 2 above)
Comment (johnduffell) [reply project/Dependencies.scala:22]: this is out on maven central now https://mvnrepository.com/artifact/com.gu/kinesis-logback-appender

**Review by kelvin-chappell — APPROVED** [review]
Great work!  Would it be possible to make clear which dependencies are there to satisfy Snyk?  So that we know easily what can be removed when the direct dependencies aren't pulling in vulnerabilities any more.  There might be other reasons why we're overriding dependencies for instance and it might not be immediately obvious.

**Comment by johnduffell** [comment]
Thanks for the approval @kelvin-chappell and could you be a bit more specific as the only one I can see is this one and should it be more clear perhaps?  Feel free to make a suggestion or even just a fresh PR and I'm happy to merge it.
```scala
"com.google.guava" % "guava" % "30.1.1-jre", // until https://github.com/playframework/playframework/pull/10874
```

**Pull request merged**